tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -24,8 +24,17 @@ sub run {
     @ret = qesap_execute(cmd => 'ansible', cmd_options => '--profile', verbose => 1, timeout => 3600);
     if ($ret[0])
     {
-        my $rec_timeout = qesap_ansible_log_find_timeout($ret[1]);
-        if ($rec_timeout) {
+        if (qesap_ansible_log_find_missing_sudo_password($ret[1])) {
+            record_info('DETECTED ANSIBLE MISSING SUDO PASSWORD ERROR');
+            @ret = qesap_execute(cmd => 'ansible', cmd_options => '--profile', verbose => 1, timeout => 3600);
```

Comment:
Running again Ansible for the second time has been proof not to be OK, at least was not ok when we was using it for `find_timeout` here below. Please look at this sequence of PR:
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17427 create `qesap_ansible_log_find_timeout` and call it but without to attempt an Ansible re-execution (sort of prparation PR)
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17505 : call Ansible for the second time. 
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17542 This one is mostly just about managing result also for the retry. This PR is very similar to this one, but it was not enough so we went further ...
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17643 final implementation is, in case of failure, to retry the full deployment. So 1. failure 2. destroy terraform 3. second terraform apply 4. second ansible

Reply:
Though the "ansible retry" works in VR: https://openqaworker15.qa.suse.cz/tests/223965#step/deploy/48. But maybe use  the same way of "find_timeout" does is better. We can open a new ticket for this.
Thanks for the comments left in [TEAM-8479](https://jira.suse.com/browse/TEAM-8479)
qesap regression GCP 15sp5 failure in Ansible - Missing sudo password